port/aix/omrvmem.c

```diff
@@ -968,7 +970,8 @@ omrvmem_find_valid_page_size(struct OMRPortLibrary *portLibrary, uintptr_t mode,
 			}
 		}
 		if ((OMRPORT_VMEM_MEMORY_MODE_EXECUTE != (OMRPORT_VMEM_MEMORY_MODE_EXECUTE & mode)) ||
-			((TRUE == codeCacheConsolidationEnabled) && (SIXTEEN_M == validPageSize)))
+			((TRUE == codeCacheConsolidationEnabled) && (SIXTEEN_M == validPageSize)) ||
+			(SIXTY_FOUR_K == validPageSize))
 #endif /* defined(OMR_ENV_DATA64) */
 		{
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions
```suggestion
			|| ((TRUE == codeCacheConsolidationEnabled) && (SIXTEEN_M == validPageSize))
			|| (SIXTY_FOUR_K == validPageSize)
#endif /* defined(OMR_ENV_DATA64) */
		) {
```